libvast/include/vast/detail/exception_ptr.hpp

```diff
@@ -0,0 +1,32 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <exception>
+#include <string>
+
+namespace vast::detail {
+
+// Taken from https://stackoverflow.com/a/37222762/92560
+// with minor adjustments and style changes.
+inline std::string what(const std::exception_ptr& eptr) {
```

Comment:
Where is this used?

Reply:
Ah, right, it was required by an earlier version that used an async asio call to start the server; I'll remove it again from the PR.